Mage.Tests/src/test/java/org/mage/test/cards/abilities/oneshot/exile/BanishmentTest.java

```diff
@@ -0,0 +1,46 @@
+package org.mage.test.cards.abilities.oneshot.exile;
+
+import mage.constants.MultiplayerAttackOption;
+import mage.constants.PhaseStep;
+import mage.constants.RangeOfInfluence;
+import mage.constants.Zone;
+import mage.game.FreeForAll;
+import mage.game.GameException;
+import mage.game.mulligan.MulliganType;
+import org.junit.Ignore;
+import org.junit.Test;
+import org.mage.test.serverside.base.CardTestPlayerBase;
+
+public class BanishmentTest extends CardTestPlayerBase {
```

Comment:
Extend `CardTestCommander4Players` (full range of influence) or `CardTestMultiPlayerBase` (range of influence of 1, and not commander) for a multiplayer game.